compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -57,9 +57,13 @@ use std::process::{self, Command, Stdio};
 use std::str;
 use std::sync::LazyLock;
 use std::time::Instant;
+#[allow(unused_imports)]
+use std::{compile_error as print, compile_error as println};
```

Comment:
These imports need a comment about their purpose, anybody could remove them and the removal will pass CI.